include/swift/AST/Types.h

```diff
@@ -780,6 +780,9 @@ class alignas(1 << TypeAlignInBits) TypeBase {
 
   /// Check if this type is equal to Builtin.IntN.
   bool isBuiltinIntegerType(unsigned bitWidth);
+  
+  /// Check if this type is declared in the Standard Library.
```

Comment:
"Check if this is a nominal type defined at the top level of the Swift module" would be more accurate, also isStdlibDecl() could have a similar comment.

Reply:
Fixed, Thanks @slavapestov!